module/token-ruler.mjs

```diff
@@ -6,7 +6,7 @@ export default class ZoneTokenRuler extends foundry.canvas.placeables.tokens.Tok
     if (!context) return;
 
     const scene = game.scenes.current;
-    if (scene.grid.units != "zone")
+    if (scene.grid.units != game.i18n.localize("ZONE-MOVEMENT.zone"))
```

Comment:
To avoid confusion during the configuration of the module, I am considering leaving this still as "zone" and only change the zone word on the label displayed.